examples/iam-github-oidc/main.tf

```diff
@@ -46,6 +46,18 @@ module "iam_github_oidc_role" {
     "terraform-aws-modules/terraform-aws-iam:ref:refs/heads/master",
   ]
 
+  # This ensures that the OIDC token GitHub uses to assume the AWS IAM role has the correct
+  # `actor` scope. Any other GitHub OIDC claims can be used as well.
+  additional_provider_trust_policy_conditions = [
+    {
+      test     = "StringEquals"
+      variable = "${module.iam_github_oidc_provider.url}:actor"
```

Comment:
Could you please include a link to the GitHub documentation describing this feature? I didn't know about it myself :)

Reply:
Certainly! The link docs are linked in the PR description but I will add a reference to them in the example as well 👍

Reply:
updated doc comment!